releasenotes.txt

```diff
@@ -1,4 +1,155 @@
-﻿Build 5.0.3
+﻿Build 5.1.0
+=============================
+
+Release notes - NHibernate - Version 5.1.0
+
+** Highlights
+    * NHibernate has gained two new target frameworks: .Net Core 2.0 and .Net Standard 2.0. NHibernate NuGet package
+      provides them, along with the .Net framework 4.6.1 build.
```

Comment:
Need to mention limitations: 

1. Binary serialization is not supported - the user shall implement [serialization surrogates](https://docs.microsoft.com/en-us/dotnet/api/system.runtime.serialization.iserializationsurrogate) for `System.Type`, `FieldInfo`, `PropertyInfo`, `MethodInfo`, `ConstructorInfo`, `Delegate`, etc. Also other limitations mentioned in the PR.